ur_controllers/src/passthrough_trajectory_controller.cpp

```diff
@@ -233,17 +241,23 @@ controller_interface::return_type PassthroughTrajectoryController::update(const
   /* Keep track of how long the trajectory has been executing, if it takes too long, send a warning. */
   if (command_interfaces_[CommandInterfaces::PASSTHROUGH_TRAJECTORY_TRANSFER_STATE].get_value() == 4.0) {
     scaling_factor_ = state_interfaces_[StateInterfaces::SPEED_SCALING_FACTOR].get_value();
+
     active_trajectory_elapsed_time_ += static_cast<double>(scaling_factor_ * (period_ns / pow(10, 9)));
+
     if (active_trajectory_elapsed_time_ > (max_trajectory_time_ + goal_time_tolerance_.seconds()) &&
         trajectory_active_) {
       RCLCPP_WARN(get_node()->get_logger(), "Trajectory should be finished by now. You may want to cancel this goal, "
                                             "if it is not.");
       trajectory_active_ = false;
     }
   }
+  AsyncInfo info = { command_interfaces_[CommandInterfaces::PASSTHROUGH_TRAJECTORY_TRANSFER_STATE].get_value(),
+                     command_interfaces_[PASSTHROUGH_TRAJECTORY_ABORT].get_value(),
+                     state_interfaces_[StateInterfaces::CONTROLLER_RUNNING].get_value(), true };
```

Comment:
Why do we need to get this information from the hardware interface? The controller should know by itself whether it is running or not?